recaf-ui/src/main/java/me/coley/recaf/ui/pane/ScriptManagerPane.java

```diff
@@ -0,0 +1,235 @@
+package me.coley.recaf.ui.pane;
+
+import javafx.application.Platform;
+import javafx.geometry.Insets;
+import javafx.geometry.Orientation;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.*;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import me.coley.recaf.RecafUI;
+import me.coley.recaf.scripting.Script;
+import me.coley.recaf.scripting.ScriptResult;
+import me.coley.recaf.ui.docking.DockTab;
+import me.coley.recaf.ui.docking.RecafDockingManager;
+import me.coley.recaf.ui.util.Animations;
+import me.coley.recaf.ui.util.Icons;
+import me.coley.recaf.ui.util.Lang;
+import me.coley.recaf.ui.window.MainMenu;
+import me.coley.recaf.util.Directories;
+
+import java.awt.*;
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.WatchKey;
+import java.nio.file.WatchService;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import static java.nio.file.StandardWatchEventKinds.*;
+
+public class ScriptManagerPane extends BorderPane {
+    private static final ScriptManagerPane instance = new ScriptManagerPane();
+
+    private final VBox scriptsList = new VBox();
+    private final ScrollPane scrollPane = new ScrollPane(scriptsList);
+
+    private static final ExecutorService THREAD_POOL = Executors.newSingleThreadExecutor((runnable) -> {
+        Thread t = new Thread(runnable);
+        t.setDaemon(true);
+        return t;
+    });
+
+    // Set up a watcher service to monitor changes in the scripts directory
+    static {
+        THREAD_POOL.submit(() -> {
+            try (WatchService watchService = FileSystems.getDefault().newWatchService()) {
+                Directories.getScriptsDirectory().register(watchService, ENTRY_MODIFY, ENTRY_DELETE);
+                while (!Thread.interrupted()) {
+                    WatchKey wk = watchService.take();
+                    if (!wk.pollEvents().isEmpty())
+                        Platform.runLater(instance::populateScripts);
+                    if (!wk.reset())
+                        break;
+                }
+            } catch (Exception ex) {
+                ex.printStackTrace();
+            }
+        });
+    }
+
+    // No public construction
+    private ScriptManagerPane() {
+        scrollPane.setFitToWidth(true);
+        scrollPane.setHbarPolicy(ScrollPane.ScrollBarPolicy.NEVER);
+
+        setCenter(scrollPane);
+
+        BorderPane actionPane = new BorderPane();
+        actionPane.setPadding(new Insets(2, 2, 2, 2));
+
+        HBox left = new HBox();
+        left.setSpacing(4);
+
+        Button newScript = new Button(Lang.get("menu.scripting.new"), Icons.getIconView(Icons.PLUS));
```

Comment:
Use `Lang.getBinding` for all text so that it remains consistent when language is changed, `Lang.get` should only be used for these cases where you can't pass bindings